tasks/install_nodejs.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Install Node.js lts
+  shell: ". ./nvm.sh && nvm install --lts"
+  args:
+    chdir: "{{ nvm_install_path }}"
```

Comment:
Unfortunately this `chdir` doesn't fix the issue I'm aware of, but if it helps in your workflow, let's do it as it doesn't seem to break anything!